protocols/smtp/src/main/java/org/apache/james/protocols/smtp/core/AbstractSenderAuthIdentifyVerificationRcptHook.java

```diff
@@ -68,8 +60,14 @@ public HookResult doRcpt(SMTPSession session, MailAddress sender,
         }
         return HookResult.declined();
     }
-    
-    
+
+    public String retrieveSender(MailAddress sender, MailAddress senderAddress) {
```

Comment:
I don't understand this signature: what is the difference between a MailAddress sender and a MailAddress senderAddress?

Reply:
Me neither, but I still believe it is more readable being extracted (it avoided re-allocation).